data/human/free worlds 1 start.txt

```diff
@@ -0,0 +1,2228 @@
+	
+	npc
+		government "Free Worlds"
+		personality heroic disables staying
+		system "Kornephoros"
+		fleet
+			names "free worlds capital"
+			fighters "free worlds fighters"
+			variant
+				"Falcon" 2
+				"Osprey" 2
+			variant
+				"Bastion"
+				"Argosy (Missile)" 2
+			variant
+				"Argosy (Blaster)"
+				"Hawk" 2
+				"Hawk (Rocket)" 2
+	npc
+		government "Free Worlds"
+		personality heroic disables staying derelict
+		system "Kornephoros"
+		fleet
+			names "free worlds capital"
+			fighters "free worlds fighters"
+			variant
+				"Falcon"
+				"Osprey"
+				"Sparrow" 3
+	npc evade
+		government "Republic"
+		personality heroic staying
+		system "Kornephoros"
+		fleet
+			names "republic capital"
+			fighters "republic fighter"
+			variant
+				"Protector (Quad Blaster)"
+				"Frigate" 2
+	npc evade
+		government "Republic"
+		personality heroic staying derelict
+		system "Kornephoros"
+		fleet
+			names "republic capital"
+			fighters "republic fighter"
+			variant
+				"Cruiser"
+				"Gunboat"
+				"Rainmaker"
+	
+	on visit
+		dialog `You've landed on <planet>, but you haven't yet completed all the objectives. Either there are still Navy ships in the system, or one of your ships carrying JJ is not yet in the system.`
+
+
+
+mission "FW Early Warning 4"
+	landing
+	name "Preemptive Strike"
+	description "Meet the Navy ship for battle in <system>, attacking them before they can enter Free Worlds space. Once all the ships have been destroyed, land on <planet>."
+	source "Deep"
+	destination "Hope"
+	to offer
+		has "FW Early Warning 3: done"
+	on fail
+		dialog `You have failed an essential Free Worlds mission. If you want to complete the story line, revert to the autosave or another earlier snapshot of the game.`
+	
+	on offer
+		event "battle in Wei"
+		conversation
+			`When you return, Tomek is busy preparing the fleet. It looks like most of the ships are ready to go, with some already starting their engines. "I take it the peace talks didn't work out," he says.`
+			`	"Not quite," says JJ. "Although I think we won a small victory in another way."`
+			`	"You'll have to tell me later," Tomek says, beginning to shout as ships around you begin to take off. "Captain <last>, get your ship ready for the air. Our sensors tell us there's a massive Navy fleet in <system>. They could jump in here any hour now. I've ordered a preemptive strike; we're going to rush into <system> and take them head on before they can reach our territory. Detain any surviving Navy personnel on <planet> and we'll send in transports to either ferry them back to the Republic if they agree to join the Oathkeepers, or to Bourne if they refuse. Now get going. We have no time to waste."`
+				accept
+	
+	npc
+		government "Free Worlds"
+		personality heroic disables
+		fleet
+			names "free worlds capital"
+			variant
+				"Falcon"
+				"Osprey" 2
+			variant
+				"Bastion"
+				"Argosy (Missile)" 2
+			variant
+				"Argosy (Blaster)"
+				"Hawk" 2
+				"Hawk (Rocket)"
```

Comment:
Will this spawn all three variants or just one?

Reply:
This only picks one. But the intention was to spawn all three. But this was copy and pasted from the "Liberate Kornephoros" mission, which I didn't touch the fleets for. And checking the blame, no one has ever touched those fleets since they were added, which means for the past 8 years only a single fleet in Liberate Kornephoros has been randomized with everything else being set in stone. :james: